docs/tests/t011.md

```diff
@@ -0,0 +1,344 @@
+1. verify that the form **Confirm Reassign Study Subject** opens
+1. click **Submit** to confirm
+
+**expected results:**   
+The message **The Subject TEST01 has been reassigned to Study/Site Site A.** appears.
+Click on **Show More** and verify that the subject is now assigned to the site.
+
+## T011-14: schedule event
+**description:**  
+This test focuses on scheduling an event for a subject.  
+
+**prerequisite:**  
+You must be logged in to a study that has at least one subject with at least one event that has not been scheduled.
+
+**workflow:**  
+1. click in the main menu on **Subject Matrix**
+1. click on the icon representing the un-scheduled event and select option **Schedule**
+1. verify that form **Schedule Study Event for XXX** opens
+1. click **Proceed to Enter Data**
+
+**expected results:**   
+The message **The study event with definition 'Rep_Null' and subject 'TEST01' was created successfully.** appears.
+Verify that in the subject matrix the status of this event has changed to **Scheduled**.
+
+## T011-15: view / enter data
+**description:**  
+This test focuses on starting to view or enter data in an event, using the subject matrix.  
+
+**prerequisite:**  
+You must be logged in to a study that has at least one subject with at least one event with status scheduled.
+
+**workflow:**  
+1. click in the main menu on **Subject Matrix**
+1. click on the icon representing the scheduled event and select option **View/Enter Data**
+
+**expected results:**   
+The form **Enter or Validate Data for CRFs in XX** opens.
+
+## T011-16: edit event
+**description:**  
+This test focuses on editing the properties of an event for a subject.  
+
+**prerequisite:**  
+You must be logged in to a study that has at least one subject with at least one event with status scheduled.
+
+**workflow:**  
+1. click in the main menu on **Subject Matrix**
+1. click on the icon representing the scheduled event and select option **Edit**
+1. verify that form **Update Study Event** opens
+1. change the start date and the status of the event
+1. click **Submit Changes**
+
+**expected results:**   
+The message **The Study Event was updated successfully.** appears.
+In the form **Enter or Validate Data for CRFs in XXX**, verify that the changes were saved.
+
+## T011-17: remove event
+**description:**  
+This test focuses on removing an event for a subject. 
+
+**prerequisite:**  
+You must be logged in to a study that has at least one subject with at least one event with data.
+
+**workflow:**  
+1. click in the main menu on **Subject Matrix**
+1. click on the icon representing the scheduled event and select option **Remove**
+1. verify that form **Remove Event from Study** opens
+1. click **Remove Event from Study**
+
+**expected results:**   
+The message **The Event XX has been removed from the Subject Record for YY in the Study ZZ.** appears.
+In the form **View Subject: YY**, verify that the status of the crf's is **Invalid**.
```

Comment:
Should be moved to T013 study events.
Thereby we also get rid of placeholders ;)